Comithook und Textfixes

Xaratas opened the pull request:
Der Pre Commit Hook macht ohne das verschieben in .git/hooks nichts.
Liegt jetzt mit dabei, weil ich den in meinem master Zweig drin habe …

Xaratas committed:
Einzelteile für einen pre-commit hook.
pre-commit muss nach .git/hooks kopiert werden, siehe readme

Xaratas committed:
Merge remote-tracking branch 'upstream/master'

Xaratas committed:
Merge remote-tracking branch 'upstream/master'

Xaratas committed:
Merge remote-tracking branch 'upstream/master'

Xaratas committed:
Es ist doch immernoch was übrig.

phob commented:
Bitte die exclude.lst erweitern die an /bin/zip weitergereicht wird, denn das hat nichts in der Mod zu suchen.

Xaratas committed:
Eine Stelle überschrieben, da das auf die Ingamesituation besser passt.
Der (fast) Ritter vom Schmelztiegelorden wird kaum zum gehen zu bewegen sein.

Merge remote-tracking branch 'upstream/master'

Conflicts:
	text/conversations/01_defiance_bay_copperlane/01_cv_doran.stringtable

Xaratas committed:
Commithook in die Hilfsdatei fürs zippen aufgenommen.

Xaratas commented:
Interessant. Das Pullrequest aktualisiert sich von alleine.

Xaratas commented:
Sorry, lib ganz übersehen. Ewig auf die Dateien gestarrt und den Ordner nicht bemerkt.

phob commented:
Hab es auch erst in der zip gesehen. Alles gut :)

Am 07.04.15 schrieb Xaratas notifications@github.com:

> Sorry, lib ganz übersehen. Ewig auf die Dateien gestarrt und den Ordner
> nicht bemerkt.
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/phob/pillarsofeternity-german-patch/pull/66#issuecomment-90729846
